maro/rl/actor/abs_actor.py

```diff
@@ -9,33 +9,36 @@
 
 
 class AbsActor(ABC):
-    def __init__(self, env: Env, inference_agents: Union[dict, AbsAgentManager]):
-        """
-        Actor contains env and agents, and it is responsible for collecting experience from the interaction
-        with the environment.
+    def __init__(self, env: Env, inference_agents: Union[AbsAgentManager, dict]):
+        """Abstract actor class.
+
```

Comment:
Better to be the summary of this class, not only for this __init__ function. Same change to other class/__init__